satellite/payments/clearing.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package payments
+
+import "context"
+
+// Clearing runs process of reconciling transactions deposits,
+// customer balance, invoices and usages.
+type Clearing interface {
```

Comment:
I'm not quite following why is this interface necessary?

Reply:
to provide mock implementation for storj-sim and tests

Reply:
I wonder whether mocking the `coinpayments` server would be a better option rather than the client?

Reply:
It should be, but as we don't have any good implementation of mocked server for `stripe` nor `coinpayments` that would significantly increase development time

Reply:
There exists https://github.com/stripe/stripe-mock, but yeah... sure let's go with mocking the client side for now.

Reply:
that has super poor functionality, and that is for `stripe` only we need `coinpayments` as well. Also mocked server is better only for tests of the `stripecoinpayments` implementation itself(benefit is limited as we won't get real world scenario, and we don't need to test API client itself), other `satellite` parts doesn't really care where data come from.